src/main/resources/db/migration/V1_14__add_hmp_fosse_way.sql

```diff
@@ -0,0 +1,4 @@
+`INSERT INTO PRISON (PRISON_ID, NAME, ACTIVE) VALUES ('FEI', 'Fosse Way (HMP/YOI)', false);
```

Comment:
We don't know if the full address is the same as _Glen Parva_ so not including one for now